native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/tcp-native/reflect-config.json

```diff
@@ -0,0 +1,179 @@
+[
+  {
+    "name":"io.ballerina.stdlib.tcp.TcpClient$1",
+    "queriedMethods":[{"name":"exceptionCaught","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Throwable"] }]
+  },
+  {
+    "name":"[Lio.ballerina.runtime.api.types.ResourceMethodType;"
```

Comment:
We may not need to explicitly register these classes for reflection. There is a chance where these are registered automatically by the native-image tool. So can we try to remove these classes and test again? It is recommended to have the minimal configurations.